validators/conversation.go

```diff
@@ -0,0 +1,43 @@
+package validators
+
+import (
+	conversationsv1 "accounts-service/protorepo/noted/accounts/v1"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+)
+
+func ValidateCreateConversationRequest(in *conversationsv1.CreateConversationRequest) error {
+	return validation.ValidateStruct(in,
+		validation.Field(&in.GroupId, validation.Required, is.UUID),
+		validation.Field(&in.Title, validation.Required, validation.Length(1, 20)),
+	)
+}
+
+func ValidateGetConversationRequest(in *conversationsv1.GetConversationRequest) error {
+	return validation.Validate(in.ConversationId,
+		validation.Required,
+		is.UUID,
+	)
+}
+
+func ValidateUpdateConversationRequest(in *conversationsv1.UpdateConversationRequest) error {
+	return validation.ValidateStruct(in,
+		validation.Field(&in.ConversationId, validation.Required, is.UUID),
+		validation.Field(&in.Title, validation.Required, validation.Length(1, 30)),
```

Comment:
When creating the conversation the character limit on the title is 20. But when you update it's 30? Also in the future, it would be cool to define our own validation rules. For example `is.ConversationTitle`.

Reply:
Okay good idea, i will see how to do that. But like for CreateConversation, isn't it better to put the TitleLength field on the ```UpdateConversationRequest``` in the protobuf?